dev-utils/win_installer/_base.sh

```diff
@@ -111,7 +112,7 @@ flake8==3.9.2
 "
 
     build_pip install --no-binary ":all:" \
-        --force-reinstall $(echo "$PIP_REQUIREMENTS" | tr ["\\n"] [" "])
+        --force-reinstall "$(echo "$PIP_REQUIREMENTS" | tr "\\n" " ")"
```

Comment:
that's not the same, the lines need to be passed as separate arguments

Reply:
Oh do you mean the outer quotes?

Reply:
Yes

Reply:
Fixed, thanks!